src/services/fetch-utils.js

```diff
@@ -0,0 +1,21 @@
+import { client, checkError } from './client';
+
+export async function getPokemon(from = 0, to = 30) {
+  const response = await client
+    .from('pokemon')
+    .select()
+    .range(from, to);
+
+  return checkError(response);
+
+
+}
+
+export async function getSinglePokemon({ id }) {
```

Comment:
Here `id` is just an argument you are passing into `getSinglePokemon` Try removing the `{}` from around `id` in your declaration. You can leave them in your match statement.